### DISCUSSION TITLE
[STO-835] Invalid request: `databases` should NOT have more than 10 items.

### DISCUSSION THREAD

**Discussion by NachoPorrasLGFM** [vercel/community#3649] [Feature Question]
### Question

Hi i was integrating prisma on my proyect, when i used the comand npx prisma migrate dev, this command generated a error. This error have created prisma_migrate_shadow databases and now i can't visit the  database's setting on vercel dashboard.

![image](https://github.com/vercel/community/assets/89904870/0357ff94-eecd-4aa7-b606-48f4a4563ad4)

**Comment by swarnava** [comment]
Hi,

Are you able to access this from local setup?

**Reply by NachoPorrasLGFM** [reply]
Hi, i can access with pgAdmin4 bit i need super user permission to delete this database

**Reply by swarnava** [reply]
I mean can you access from your next js application by serving it from localhost?

**Reply by NachoPorrasLGFM** [reply]
sorry,  no i don't can access on localhost to vercel storage. i don't can create tables or import data with prisma now on vercel storage.
I have could create a new database on my localhost database server with prisma and this is working

**Reply by swarnava** [reply]
Thank you! We are investigating this currently and it seems like an issue related to Prisma. Unfortunately we don't have an ETA for this.
